### ISSUE TITLE
Please Help Me. Can i use Retrieve Pointer to User Space (QUSPTRUS) API 

### ISSUE THREAD

**Issue by maxkub47** [IBM/nodejs-itoolkit#406]
![image](https://github.com/user-attachments/assets/159f6a4a-f6b9-4364-bd63-6e3dec2dfeb4)

"Since this program needs to return a parameter of type PTR(SPP), how can I write this in Node.js? Do you have any examples?"

**Comment by abmusse** [comment]
Are you trying to just retrieve data from the user space?

If so there other ways to do so without getting back a pointer:

https://www.ibm.com/docs/en/i/7.3?topic=apis-manipulating-user-space-without-pointers

There is a function in this library to get user space data via QUSRTVUS see https://nodejs-itoolkit.readthedocs.io/en/latest/deprecated/iUserSpace.html#iUserSpace.getUserSpaceData

**Comment by kadler** [comment]
There's no way to retrieve pointers through XMLSERVICE as the pointer tags get unset, even if the bytes get preserved. The workaround that @abmusse mentioned using QUSRTVUS API is the way to go.

**Comment by maxkub47** [comment]
Thank you very much for the advice.

Previously, I tried using the USER_SPACE table function 
`SELECT * FROM TABLE(QSYS2.USER_SPACE(
                                               USER_SPACE => 'USRSPACE1', 
                                               USER_SPACE_LIBRARY => 'TESTLIB'));`
to directly extract data, but I encountered unreadable values mixed in, making it difficult to pinpoint the desired value.

I will now try coding the program based on your advice to see if the result differs from the table function.
